### PR TITLE
IE tweaks to improve display

### PR DESCRIPTION
**Pull request by bmansheim**

### CODE REVIEW

**Comment by bmansheim** [comment]
@HarunD I noticed that:

1. The Redis logo is smaller than in Chrome
1. The version selector is missing
1. Code boxes don't reach across the page like in Chrome.
  For example, https://docs.redislabs.com/staging/ie-tweaks/rs/getting-started/quick-setup/
1. Issue with page width (images, notes, etc) :
  https://docs.redislabs.com/staging/ie-tweaks/rs/getting-started/quick-setup/
  https://docs.redislabs.com/staging/ie-tweaks/rs/concepts/kubernetes-architecture/redis-labs-kubernetes-architecture-overview/
  https://docs.redislabs.com/staging/ie-tweaks/rs/faqs/
  https://docs.redislabs.com/staging/ie-tweaks/rs/developing/crdbs/developing-hashes-crdb/
1. RS link on top in RC - https://docs.redislabs.com/staging/ie-tweaks/rc/
1. RS and RC links on top in RV - https://docs.redislabs.com/staging/ie-tweaks/rv/
1. Needs more padding on right - https://docs.redislabs.com/staging/ie-tweaks/rv/administration/setup_and_editing/view-edit-cloud-account/

I tried to cover a wide range of pages so I hope that's it.

**Comment by HarunD** [comment]
@bmansheim I missed these, thanks for the feedback. I will fix those in the coming days.

File: static/theme-flex/style.css [hunk 1]
```diff
@@ -2662,7 +2662,7 @@ a[href*="#no-click"], img[src*="#no-click"] {
         flex: 1 0 auto;
     }
     article section.page .main-content .highlight {
-        display: block;
+        /* display: block; */
```
Comment (bmansheim) [inline static/theme-flex/style.css]: @HarunD I looked at the staging site in IE and it looks great. I just found that links are pushed onto their own line. This change fixed it. What do you think?

Comment (HarunD) [reply static/theme-flex/style.css]: This change looks good, it seems `display: block` is not needed.

File: static/theme-flex/style.css [hunk 2]
```diff
@@ -2692,7 +2692,7 @@ a[href*="#no-click"], img[src*="#no-click"] {
     article section.page p>code,
     article section.page li code,
     article section.page table code {
-        display: block;
+        /* display: block; */
```
Comment (bmansheim) [inline static/theme-flex/style.css]: This was making inline code text move to its own line. You can see it at `/rs/installing-upgrading/downloading-installing/#prerequisites` in the expanding block.
Is it safe to make this change?

Comment (HarunD) [reply static/theme-flex/style.css]: I checked it, the change is safe to make.

**Pull request merged**